public/icons.svg

```diff
@@ -0,0 +1 @@
6.204,39,35c0-3.859-3.14-7-7-7s-7,3.141-7,7 s3.14,7,7,7c1.204,0,2.338-0.306,3.329-0.844l5.257,5.258L43.414,43.586z M27,35c0-2.757,2.243-5,5-5s5,2.243,5,5s-2.243,5-5,5 S27,37.757,27,35z"/></g></symbol><symbol id="nc-icon-f-check-glyph-48" viewBox="0 0 48 48"><title>f check</title><g transform="translate(0, 0)"><polygon points="18 40 3 25 8 20 18 30 40 8 45 13 18 40" fill="currentColor"/></g></symbol><symbol id="nc-icon-camera-2-glyph-48" viewBox="0 0 48 48"><title>camera 2</title><g transform="translate(0, 0)"><path data-color="color-2" d="M9,10H6a5.006,5.006,0,0,0-5,5V39a5.006,5.006,0,0,0,5,5H9Z"/><path d="M42,10H37.58L33.868,3.5A1,1,0,0,0,33,3H21a1,1,0,0,0-.868.5L16.42,10H11V44H42a5.006,5.006,0,0,0,5-5V15A5.006,5.006,0,0,0,42,10ZM27,37A10,10,0,1,1,37,27,10,10,0,0,1,27,37Z" fill="currentColor"/></g></symbol><symbol id="nc-icon-s-question-glyph-48" viewBox="0 0 48 48"><title>s question</title><g transform="translate(0, 0)"><path d="M42,3H6A3,3,0,0,0,3,6V42a3,3,0,0,0,3,3H42a3,3,0,0,0,3-3V6A3,3,0,0,0,42,3ZM23,38a2,2,0,1,1,2-2A2,2,0,0,1,23,38Zm8.829-19.4a7.39,7.39,0,0,1-1.123,2.05,27.036,27.036,0,0,1-3.2,3.116A12.018,12.018,0,0,0,24.9,26.558a5.979,5.979,0,0,0-.635,2.968v.645H21.741V29.116a8.749,8.749,0,0,1,.7-3.76,10.076,10.076,0,0,1,2.617-3.115,17.684,17.684,0,0,0,3.35-3.379,5.142,5.142,0,0,0,.693-2.734,3.811,3.811,0,0,0-1.279-3.076,5.544,5.544,0,0,0-3.682-1.084,12.509,12.509,0,0,0-3.008.361,16.381,16.381,0,0,0-3.359,1.318l-1.152-2.636a16.411,16.411,0,0,1,7.715-1.934,8.447,8.447,0,0,1,5.8,1.836,6.556,6.556,0,0,1,2.071,5.176A7.564,7.564,0,0,1,31.829,18.6Z" fill="currentColor"/></g></symbol><symbol id="nc-icon-a-search-glyph-48" viewBox="0 0 48 48"><title>a search</title><g transform="translate(0, 0)"><path fill="currentColor" d="M22,27c6.195,0,11-6.988,11-13v-2c0-6.065-4.935-11-11-11S11,5.935,11,12v2C11,20.012,15.805,27,22,27z"/><path data-color="color-2" d="M46.707,45.293l-5.109-5.108C42.474,39.016,43,37.57,43,36c0-3.86-3.141-7-7-7s-7,3.14-7,7 s3.141,7,7,7c1.57,0,3.015-0.526,4.184-1.401l5.109,5.109C45.488,46.902,45.744,47,46,47s0.512-0.098,0.707-0.293 C47.098,46.316,47.098,45.684,46.707,45.293z M31,36c0-2.757,2.243-5,5-5s5,2.243,5,5s-2.243,5-5,5S31,38.757,31,36z"/><path fill="currentColor" d="M27,36c0-2.097,0.727-4.024,1.934-5.556C26.824,30.174,24.5,30,22,30c-6.553,0-11.925,1.188-15.278,2.185 C3.3,33.203,1,36.286,1,39.856V44c0,0.552,0.447,1,1,1h34C31.037,45,27,40.962,27,36z"/></g></symbol><symbol id="nc-icon-face-man-glyph-48" viewBox="0 0 48 48"><title>face man</title><g transform="translate(0, 0)"><path fill="currentColor" d="M24,1C11.318,1,1,11.318,1,24s10.318,23,23,23s23-10.318,23-23S36.682,1,24,1z M24,45C12.42,45,3,35.58,3,24 c0-0.595,0.041-1.181,0.089-1.764c2.658-3.33,6.237-3.336,10.744-3.334c5.119,0.012,11.45,0.007,17.959-5.399 c4.957,7.216,10.906,8.954,13.15,9.364C44.963,23.245,45,23.618,45,24C45,35.58,35.58,45,24,45z"/><circle data-color="color-2" cx="15" cy="29" r="3"/><circle data-color="color-2" cx="33" cy="29" r="3"/></g></symbol></svg>
```

Comment:
I’m not sure about this, but it might be possible to import the SVG file in one of the components (or rather in a separate *icon*  component. That way, we’d get hashed file names for this too. That would require that webpacker has an SVG loader setup by default, too, or we set that up ourselves. (But we don’t need to sort that out right now.)